doc/source/index.rst

```diff
@@ -16,6 +16,7 @@ Welcome to VerMoUTH's documentation!
    file_formats
    tutorials/index
    api/modules
+   gromacs_variables
```

Comment:
Maybe between file formats and the tutorials is a better spot? At least above the api docs